docs/rackhd_vlab/web_ui.rst

```diff
@@ -1,2 +1,229 @@
 RackHD try-out with Web UI
 =============================
+
+In this session, you will learn how to configure web-ui and customize a RackHD workflow to implement your own logic.
+
+Configure on-web-ui
+~~~~~~~~~~~~~~~~~~~~
+
+* Open web browser, and then go to the following URL ``http://10.62.59.196:9090/ui``, replace with your own ipaddr and port.
+* Click the ``gear`` button on the top right panel
+
+.. image:: ../_static/web-ui/s1.png
+    :align: center
+
+* Enter your ``RackHD Northbound API``, then click ``save`` button
+
+.. image:: ../_static/web-ui/s2.png
+    :align: center
+
+* Then you will see all discovered nodes in the panel.
+
+.. image:: ../_static/web-ui/s3.png
+    :align: center
+
+Workflow scenario
+~~~~~~~~~~~~~~~~~~
+
+* You have a number of new bare metal servers coming online.
+* Before the OS and applications are deployed to the new servers, you want to run a quick sanity check (diagnostic) on the servers.
+* Due to a special demand of your application, you want to include a temperature check and CPU frequency check in the diagnostic step.
+
+To fulfill the demand of scenario, you can use On-Web-UI to customize a new workflow named
+My_Workflow.
+
+This example is a simple one. However, your customized workflows can be as complex as
+needed.
+
+Workflow in RackHD
+~~~~~~~~~~~~~~~~~~~~~
+
+A workflow in RackHD is a JSON document, which describes a flow of execution and is built as a
+graph. A graph is composed by several tasks.
+The tasks can be executed in serial or in parallel. Each task has a conditional output that can be
+used to drive the workflow down different paths based on how the task is completed (for example,
+Error, Failed, Succeeded).
+
+Add a new workflow
+--------------------
+
+Go to Workflow Center -> workflow Editor
+
+.. image:: ../_static/web-ui/s18.png
+    :align: center
+
+Right click on the canvas to add tasks, type pxeboot and choose ``Task.Obm.Node.PxeBoot`` task.
+
+.. image:: ../_static/web-ui/s6.png
+    :align: center
+
+A Task block will display in the canvas as well as the json data will display on the right
+
+.. image:: ../_static/web-ui/s7.png
+    :align: center
+
+The task will be named as ``new-task-xxxxx`` (xxxxx is randomly generated), to make it more friendly you can change the label property ('set-boot-pxe' for this example) on the right panel,
```

Comment:
We have a bug here, after connect tasks, taskName can't be changed any more.
You could add a notification here.